lib/Doctrine/DBAL/Platforms/PostgreSqlPlatform.php

```diff
@@ -1054,6 +1054,14 @@ public function getReadLockSQL()
     /**
      * {@inheritDoc}
      */
+    public function getRandomFunctionSQL()
+    {
+        return 'random()';
```

Comment:
The documentation states it produces a number like this:

`random value in the range 0.0 <= x < 1.0`

Not really intended, is it?